### ISSUE TITLE
ocrmypdf doesn't find tesseract4 ghostscript unpaper qpdf

### ISSUE THREAD

**Issue by ivan** [NixOS/nixpkgs#67497]
**Describe the bug**

ocrmypdf doesn't work until I also install `tesseract4 ghostscript unpaper qpdf`.

**To Reproduce**
1. Don't have `tesseract4 ghostscript unpaper qpdf` installed or in PATH.
2. Install `ocrmypdf`
3. Run `ocrmypdf -l eng --clean-final input.pdf ocr.pdf` 
4. Observe one of:

```
# ocrmypdf -l eng --clean-final input.pdf ocr.pdf
Traceback (most recent call last):
  File "/nix/store/f3bi0cmdcwndm0vlv9xpzibv95p68rcn-ocrmypdf-8.2.3/bin/.ocrmypdf-wrapped", line 11, in <module>
    sys.exit(run_pipeline())
  File "/nix/store/f3bi0cmdcwndm0vlv9xpzibv95p68rcn-ocrmypdf-8.2.3/lib/python3.7/site-packages/ocrmypdf/__main__.py", line 1062, in run_pipeline
    check_options(options, _log)
  File "/nix/store/f3bi0cmdcwndm0vlv9xpzibv95p68rcn-ocrmypdf-8.2.3/lib/python3.7/site-packages/ocrmypdf/__main__.py", line 712, in check_options
    check_options_languages(options, log)
  File "/nix/store/f3bi0cmdcwndm0vlv9xpzibv95p68rcn-ocrmypdf-8.2.3/lib/python3.7/site-packages/ocrmypdf/__main__.py", line 529, in check_options_languages
    if not languages.issubset(tesseract.languages()):
  File "/nix/store/f3bi0cmdcwndm0vlv9xpzibv95p68rcn-ocrmypdf-8.2.3/lib/python3.7/site-packages/ocrmypdf/exec/tesseract.py", line 103, in languages
    args_tess, universal_newlines=True, stdout=PIPE, stderr=STDOUT, check=True
  File "/nix/store/yglc8kgzmdaskyngv90y1mw96fl00pjy-python3-3.7.4/lib/python3.7/subprocess.py", line 472, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/nix/store/yglc8kgzmdaskyngv90y1mw96fl00pjy-python3-3.7.4/lib/python3.7/subprocess.py", line 775, in __init__
    restore_signals, start_new_session)
  File "/nix/store/yglc8kgzmdaskyngv90y1mw96fl00pjy-python3-3.7.4/lib/python3.7/subprocess.py", line 1522, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'tesseract': 'tesseract'



# ocrmypdf -l eng --clean-final input.pdf ocr.pdf
  ERROR - Could not find program 'gs' on the PATH



# ocrmypdf -l eng --clean-final input.pdf ocr.pdf
  ERROR -
The program 'unpaper' could not be executed or was not found on your
system PATH.  This program is required when you use the
['--clean, --clean-final'] arguments.  You could try omitting these arguments, or install
the package.

   INFO -
On systems with the aptitude package manager (Debian, Ubuntu), try these
commands:
    sudo apt-get update
    sudo apt-get install unpaper

On RPM-based systems (Red Hat, Fedora), search for instructions on
installing the RPM for unpaper.



# ocrmypdf -l eng --clean-final input.pdf ocr.pdf
  ERROR -
OCRmyPDF requires 'tesseract' 4.0.0 or higher.  Your system appears
to have 3.05.00.  Please update this program.

   INFO -
On systems with the aptitude package manager (Debian, Ubuntu), try these
commands:
    sudo apt-get update
    sudo apt-get install tesseract-ocr

On RPM-based systems (Red Hat, Fedora), search for instructions on
installing the RPM for tesseract.



# ocrmypdf -l eng --clean-final input.pdf ocr.pdf
  ERROR -
The program 'qpdf' could not be executed or was not found on your
system PATH.

   INFO -
On systems with the aptitude package manager (Debian, Ubuntu), try these
commands:
    sudo apt-get update
    sudo apt-get install qpdf

On RPM-based systems (Red Hat, Fedora), search for instructions on
installing the RPM for qpdf.
```

**Additional context**
Running nixpkgs master

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.67-hardened, NixOS, 19.09.git.165aa35 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2.2`
 - channels(root): `"nixos-18.09.1922.97e0d53d669"`
 - channels(at): `""`
 - channels(play): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

cc @Kiwi

**Comment by Kiwi** [comment]
I'm really swamped right now but I do need to do some updates that were requested to be in the PR before it got merged. I was half expecting someone to find problems anyway (it was really, really difficult to even get it to work at all). I'll try to get around to fixing it ASAP... which might be in a few days... if nobody does it sooner. Sorry about it not working!

**Comment by ivan** [comment]
No, thank you for packaging it! I successfully used it today :-)

**Comment by Kiwi** [comment]
This should be fixed as of the upgrade I did to 9.0.3. @ivan would you mind testing it to confirm it truly is fixed? It's in nixos-unstable now. After that we can close this! :D

**Comment by ivan** [comment]
Thanks! This does look fixed in master.
